tests/manual_test_chatbot.py

```
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# manual_test_chatbot.py
+import sys
+import os
+
+# Add the project root to Python path
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+
+from src.services.chatbot_engine import chatbot_engine
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)